src/twisted/newsfragments/9982.misc

```diff
@@ -0,0 +1 @@
+Exempt ``__repr__``s from compatibility policy.
```

Comment:
```suggestion
Exempt ``__repr__``s ``__slots__`` and other ``@attrs`` related changes from compatibility policy.
```

Reply:
`@attrs` sits at a weird point between `attrs` and `@attr.s`; I'd prefer either of those two more literal terms.

Reply:
Also there seems to be a comma missing in the enumeration.

Reply:
I do mean the new `@attrs.frozen` interface